plugin/trino-postgresql/pom.xml

```diff
@@ -162,6 +162,12 @@
             <scope>test</scope>
```

Comment:
Could you please apply https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages